src/specs.ts

```diff
@@ -9,20 +9,13 @@
  * rights grant found at http://polymer.github.io/PATENTS.txt
  */
 
-import * as fsExtra from 'fs-extra';
 import * as path from 'path';
-import * as url from 'url';
+import {URL} from 'url';
```

Comment:
URL is also a global, since Node 10, I believe

Reply:
Not available even after adding `@types/node`.